views/helpers/handlebars.js

```diff
@@ -7,6 +7,20 @@ function hbsHelpers() {
                     return "Senior Representative";
                 case "junior-rep":
                     return "Junior Representative";
+                case "president":
```

Comment:
Why not keep this in the list of roles where it was before, that way the information can be maintained in one spot rather than 2.

Reply:
im keeping this translate helper in the case that you guys would want to support different languages and we could leverage this helper to translate any error messages / static messages from a language file, which we could load into it